src/react-apps/applications/service-development/src/actions/navigationActions/navigationActionDispatcher.ts

```diff
@@ -0,0 +1,22 @@
+import { Action, ActionCreatorsMapObject, bindActionCreators } from 'redux';
+import { store } from '../../store';
+import * as NavigationActions from './actions';
+
+export interface INavigationActionDispatchers extends ActionCreatorsMapObject {
+  toggleDrawer: () => Action;
+  toggleDrawerFulfilled: () => Action;
+  toggleDrawerRejected: (error: Error) => NavigationActions.IToggleDrawerActionRejected;
+}
+
+const actions: INavigationActionDispatchers = {
+  toggleDrawer: NavigationActions.toggleDrawerAction,
+  toggleDrawerFulfilled: NavigationActions.toggleDrawerAction,
```

Comment:
`toggleDrawerFulfilled` should have the same action as `toggleDrawer`.

If you're not using a saga for this, don't have the fulfilled-action. Just the one action to toggle.

Reply:
If you're not using Sagas... do you need any Sagas at all? Is it possible to just use "plain Redux"?

Reply:
Removed.